### PR TITLE
D8 un 2444

### PR DESCRIPTION
**Pull request by soda736**

### CODE REVIEW

File: project/sites/nicybersecuritycentre/themes/nicybersecuritycentre_theme/templates/layout/page--events.html.twig [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+  {{ page.slideshow }}
+{# If the homepage has no basic page then make the content the Main section and add the h1. #}
+{% if basic_page_structure is empty %}
+<main id="content">
+  {% if drupal_config('system.site', 'frontpage_title') %}
+    <h1 class="{{ drupal_config('system.site', 'frontpage_show_title') ? '' : 'visually-hidden' }}">
+      {{ drupal_config('system.site', 'frontpage_title') }}
+    </h1>
+  {% endif %}
+  {% endif %}
```
Comment (omahm) [inline project/sites/nicybersecuritycentre/themes/nicybersecuritycentre_theme/templates/layout/page--events.html.twig:98]: Slight nitpick here. It'd be easier to read if this was aligned to its matching 'if' statement. I checked the number of if's versus endif just to check this wasn't a duplicate.

**Review by omahm — CHANGES_REQUESTED** [review]
Looks good. Just that stacked 'endif' that caught my attention.

**Comment by soda736** [comment]
Alot of changes since the last review. This still needs a bit of tweaking but may need merged in before the D10 upgrade on Unity 2 starts.

**Review by DuttonMa — APPROVED** [review]

**Review by omahm — APPROVED** [review]
LGTM

**Pull request closed without merging**